frontend/styles/main.scss

```diff
@@ -51,6 +51,10 @@
   --flathub-celestial-blue: 74 144 217;
   --flathub-electric-red: 226 0 0;
 
+  --flathub-status-green: 67 160 71;
```

Comment:
@JakobDev could you come up with some status colors? I went for some muted ones, but you will probably know better (and the red is just the one we had before)

Reply:
Sorry to disappoint you, but I'm not really good at designing, so I can't really help you with share of green would look the best here.

Reply:
sorry @JakobDev I meant to ping @jimmac  :)

Reply:
I don't see why not embrace the low hanging fruit -- just do what libadwaita does. Sadly light/dark situation is throwing a wrench in the simple approach. CSS doesn't yet have a reasonable way to do some SASS-styled color mixing, so I've seen people using HSL definitions using custom properties for the individual components and then using calc() to tint the lightness for the dark variant. But even libadwaita just hardcodes the light/dark variants:

```
/* libadwaita colors */
$success: #2ec27e;
$success-dark: #26a269;
$warning: #e5a50a;
$warning-dark: #cd9309;
$error: #e01b24;
$error-dark: #c01c28;
```